### DISCUSSION TITLE
Adding users interactive system. (like, dislike and ... )

### DISCUSSION THREAD

**Discussion by TorhamDev** [collove/pasteme#53] [Ideas]
Hello Sadra( @lnxpy ). Let me see what you think about allowing users to rate a code snippet  like : dislike, like, emoji ....?

And users may even leave comments for code snippets 

or something cool, like real-time chat :)

**Comment by lnxpy** [comment]
That looks a bit out of business at the moment IMO! Let's see what @Farhaduneci got as well. :)
Thanks for reaching out by the way.

**Reply by farhaduneci** [reply]
Thank you for reaching me out, Sadra :crossed_fingers:

**Comment by farhaduneci** [comment]
Hey there, I appreciate @TorhamDev for this great recommendation, I follow Sadra's point of view at this moment, This feature will not fit in `pastebin` websites! People usually paste codes here to quickly share it with **their own audience**, not everyone.

> This approuch seems kinda Stack Overflow-ish to me, and well [Stack Overflow](https://stackoverflow.com) has been very successful on this.

**Reply by lnxpy** [reply]
I do agree.
